src/python/pants/task/task.py

```diff
@@ -74,6 +74,11 @@ def implementation_version(cls):
   def implementation_version_str(cls):
     return '.'.join(['_'.join(map(str, x)) for x in cls.implementation_version()])
 
+  @classmethod
+  @memoized_method
+  def implementation_version_slug(cls):
+    return sha1(cls.implementation_version_str().encode('utf-8')).hexdigest()[:12]
```

Comment:
nit: can we use the same language (e.g. `implementation_version_sha`) to keep the meaning consistent? 

https://github.com/pantsbuild/pants/blob/aad3a3caf0a01c7daeda5822b3f2661de7475379/src/python/pants/invalidation/cache_manager.py#L289-L291

Reply:
I think that this is consistent in the diff? the line you commented on is older.

Reply:
Ah good. thanks!